recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-graphics/gles-module/gles-module-egl-headers_1.10.bbappend

```diff
@@ -0,0 +1,14 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
+
+BRANCH = "1.10/ED5187610"
```

Comment:
Is there any other difference of this bbappend from the corresponding one of version 1.9 other than version?
Can we have a common inc here, so 1.9 and 1.10 only differ in that?

Reply:
Done